Makefile

```diff
@@ -114,6 +120,13 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
+.PHONY: docker-buildx-builder
+docker-buildx-builder:
+	@if ! docker buildx ls | grep $(BUILDX_BUILDER_NAME); then \
+		docker buildx create --name $(BUILDX_BUILDER_NAME) --platform windows/amd64 --use; \
```

Comment:
i think we can skip `--platform windows/amd64` here but @cpuguy83 can correct me

Reply:
Hmm, ok. Local testing kept throwing an "os unsupported" error w/o that. I updated anyway in case my error was specific to my local env.